src/actors/symbolication.rs

```diff
@@ -1304,21 +1304,28 @@ impl SymbolicationActor {
                                     object_info_from_minidump_module(object_type, code_module)
                                         .into();
 
-                                let status = unwind_statuses
-                                    .get(&code_module.id()?)
-                                    .cloned()
-                                    .unwrap_or(ObjectFileStatus::Unused);
+                                let module_id: Option<CodeModuleId> = code_module.id();
+
+                                let status = match module_id {
+                                    Some(id) => unwind_statuses
+                                        .get(&id)
+                                        .cloned()
+                                        .unwrap_or(ObjectFileStatus::Unused),
+                                    None => ObjectFileStatus::Malformed,
+                                };
 
                                 metric!(
                                     counter("symbolication.unwind_status") += 1,
                                     "status" => status.name()
                                 );
                                 info.unwind_status = Some(status);
 
-                                let features = object_features
-                                    .get(&code_module.id()?)
-                                    .copied()
-                                    .unwrap_or_default();
+                                let features = match module_id {
+                                    Some(id) => {
+                                        object_features.get(&id).copied().unwrap_or_default()
+                                    }
+                                    None => Default::default(),
+                                };
 
                                 info.features.merge(features);
 
```

Comment:
Just below here, the linter is complaining that this closure always returns `Some`, so it can be converted from `filter_map` to `map`.

Reply:
yep, forgot to run clippy locally...